### PR TITLE
Remove the instruction related to Longhorn v0.7.0

### PR DESCRIPTION
**Pull request by shuo-wu**
Since we won't support Longhorn v0.7.0 or lower versions after GA.

### CODE REVIEW

File: content/docs/0.8.1/deploy/upgrade/longhorn-manager.md [hunk 1]
```diff
@@ -26,203 +24,3 @@ helm upgrade longhorn ./longhorn/chart
...
-
-- [Prerequisites](#prerequisites)
-- [Upgrade Longhorn Manager](#upgrade-longhorn-manager)
-  - [Using the Rancher Catalog App](#using-the-rancher-catalog-app)
-  - [Using kubectl](#using-kubectl)
-- [Troubleshooting](#troubleshooting)
-- [Clean up the v0.6.2 CRDs](#clean-up-the-v062-crds)
-- [Rollbacks](#rollbacks)
-
-### Prerequisites
```
Comment (yasker) [inline content/docs/0.8.1/deploy/upgrade/longhorn-manager.md:99]: This part of content is for general upgrade steps.

**Comment by catherineluse** [comment]
So I should wait to merge it until after #122 ?

**Comment by yasker** [comment]
@catherineluse hold on merging it. It removes the manager upgrade steps.

File: content/docs/0.8.1/deploy/upgrade/longhorn-manager.md (hunk 1 above)
Comment (shuo-wu) [reply content/docs/0.8.1/deploy/upgrade/longhorn-manager.md:99]: What... I think this part is for `v0.6.2 to v0.7.0` only. See line 87:
```
You will need to follow this guide to upgrade the Longhorn manager from v0.6.2 to v0.7.0.
```

Maybe we can retain some troubleshooting guides here. But we need to make sure they are for the general cases.

Comment (yasker) [reply content/docs/0.8.1/deploy/upgrade/longhorn-manager.md:99]: The steps here has more details than the steps in `### Upgrading Longhorn Manager from v0.7.0+`.

For example, the previous one doesn't mention upgrade using Rancher catalog app or have a list of the components that suppose to be running after the upgrade.

Comment (shuo-wu) [reply content/docs/0.8.1/deploy/upgrade/longhorn-manager.md]: I saw there are some updates for this doc. I decide not to modify this part for now.

**Comment by yasker** [comment]
LGTM.

**Pull request merged**